docs/conf.py

```diff
@@ -1 +1,3 @@
 from crate.theme.rtd.conf.theme import *
+
+#suppress_warnings = ['git.too_shallow']
```

Comment:
This one will just apply to this tool repository in isolation, which is only used in development sandbox mode (`make dev`). If we want to roll out a Sphinx setting across all documentation projects, crate-docs-theme is the right place.

Reply:
I think suppressing warnings should only happen as a very last resort, of course always depending on circumstances.

Reply:
Let's remove this, merge, and then spread it out to other repositories?

Reply:
There are still a few open questions:
1. What depth is needed when fetching?
2. Do we get any value out of the new sphinx-sitemap? I.e. dates on when a page was updated? Is that for end users or search?
3. Can we disable this by configuration (assuming we don't need/want it). I remember seeing something, but don't recall exact config option right now...
4. Will this be fixed in sphinx-sitemap itself?

Therefore itt seems very attractive to be able to fix this temporarily for all docs builds just one place, so I would suggest to pin sphinx-sitemap in crate-theme instead for now.

Reply:
Hi Brian.

> It seems very attractive to be able to fix this temporarily for all docs builds just one place, so I would suggest to https://github.com/crate/crate-docs-theme/pull/611 instead for now.

That's absolutely way to go, thank you so much.